config/jobs/testing/testing-postsubmits.yaml

```diff
@@ -0,0 +1,64 @@
+presets:
+
+  - labels:
+    preset-deployer-service-account: "true"
+  env:
+  - name: GOOGLE_APPLICATION_CREDENTIALS
+    value: /creds/service-account.json
+  volumeMounts:
+    - name: creds
+      mountPath: /creds
+  volumes:
+  - name: creds
+    secret:
+      secretName: deployer-service-account
```

Comment:
This secret needs to be created & deployed still

Reply:
Happy to help with it

Reply:
Secret deployed